deluge/tests/test_security.py

```diff
@@ -23,17 +23,10 @@
 SECURITY_TESTS = bool(os.getenv('SECURITY_TESTS', False))
 
 
-@pytest.mark.skipif(windows_check(), reason='windows cannot run .sh files')
-@pytest.mark.skipif(not SECURITY_TESTS, reason='skipping security tests')
+# TODO: This whole module has not been tested since migrating tests fully to pytest
```

Comment:
We don't normally run this test as it's quite a long test but this module can be tested if you add the text security_test to PR description: https://github.com/deluge-torrent/deluge/blob/73394f1fc5da75ccf0bd466385c2141ff93bd604/.github/workflows/ci.yml#L45

Reply:
Thanks. Yep, those are looking good too.